www/gatsby-node.js

```diff
@@ -107,6 +107,9 @@ exports.onCreateWebpackConfig = ({ actions }) => {
     ],
     resolve: {
       modules: [path.resolve(__dirname, "src"), "node_modules"],
+      alias: {
+        "styled-components": require.resolve("styled-components"),
```

Comment:
@mdole - that warning you saw in the console regarding multiple instances of styled components was resolved here. What this says, basically, is when encountering 
```
import styled from 'styled-components'
```
force resolution to the closest node_modules folder, which in this case is `www/node_modules`.

This is weird webpack trickery...

Reply:
whoa that's dope. nice solution